frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/imageProperty.component.tsx

```diff
@@ -14,12 +14,35 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { TextField } from '@mui/material';
+import { useEffect } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
 import { PropertyProps } from './properties.types';
+import { TicketImage } from './basicTicketImage/ticketImage/ticketImage.component';
 
-export const ImageProperty = ({ property: { name, readOnly, required }, ...props }: PropertyProps) => (
-	<div>
-        &nbsp;
-		<TextField label={name} disabled={readOnly} required={required} {...props} />
-	</div>
-);
+export const ImageProperty = ({
+	property: { name: title, readOnly, required },
+	name,
+	defaultValue,
+	onBlur,
+	...props
+}: PropertyProps) => {
+	const { watch } = useFormContext();
+
+	useEffect(() => { onBlur?.(); }, [watch(name)]);
```

Comment:
Should be inside of TicketImage but not using react-hook-form references just 

``useffect(()=> onBlur() , [value])``